lib/plugins/aws/invokeLocal/index.js

```diff
@@ -174,7 +174,8 @@ class AwsInvokeLocal {
 
     return new BbPromise(resolve => {
       const python = spawn(runtime,
-        ['-u', path.join(__dirname, 'invoke.py'), handlerPath, handlerName], { env: process.env });
+        ['-u', path.join(__dirname, 'invoke.py'), handlerPath, handlerName],
+        { env: process.env }, { shell: true });
```

Comment:
Hi @bhaskarbalusani, Great to have done your first-time contribution to the framework and thank you for taking a look at it! 👍 Awesome!
Does this need to be added to Python as well, not just Java?

Reply:
Thank you @horike37 !! 
Yes, We need to add it for all executable's that are having extension of .cmd and .bat!!